Add OFS attribution for HSSI subsystem

Figo-zhang opened the pull request:
Adding OFS attribution for HSSI subsystem (0x15).
This is request for linux-fpga community.

Link: https://lore.kernel.org/linux-fpga/20220308064315.1452217-1-tianfei.zhang@intel.com/

Signed-off-by: Tianfei zhang <tianfei.zhang@intel.com>

fpgamatt commented on dfl-feature-ids.rst:
```diff
@@ -90,7 +90,7 @@ document.
      - 0x14
 
-   * - HSSI Subsystem
+   * - HSSI Subsystem (OFS)
```
Please change to 
OFS HSSI Subsystem

Figo-zhang replied:
i agree, i have update the patch

Figo-zhang committed:
Add OFS attribution for HSSI subsystem

Adding OFS attribution for HSSI subsystem (0x15).
This is request for linux-fpga community.

Link: https://lore.kernel.org/linux-fpga/20220308064315.1452217-1-tianfei.zhang@intel.com/

Signed-off-by: Tianfei zhang <tianfei.zhang@intel.com>
